refs/x-compile.sh

```diff
@@ -625,6 +628,70 @@ case ${KINDLE_TC} in
 
 		DEVICE_USERSTORE="/home/root"
 	;;
+	REMARKABLE_AARCH64 )
+		ARCH_FLAGS="-mcpu=coretex-a53 -march=armv8-a+crc+crypto"
```

Comment:
Unless `-mcpu` does something you really don't like and need to override, I'd prefer this the other way around (i.e., -march -> -mcpu).

Reply:
Or, given that this is arm, actually, *either*

- march -> mtune
- mcpu

Reply:
(I would tend to prefer march + mtune, because I fucking despise GCC's mcpu behavior on arm ;p)

Reply:
Keep in mind that this is all extremely nitpicky, given that the TCs are built w/ mcpu, and as such probably already do the right thing ;).